crates/frontend/src/components/default_config_form.rs

```diff
@@ -222,37 +222,37 @@ where
 
                     <option
                         value="number"
-                        selected=move || { config_type.get() == "number".to_string() }
+                        selected=move || { config_type.get() == *"number" }
```

Comment:
How does this work?

Reply:
In the first line, the literal string "number" is converted to a String object using to_string() before the comparison.
In the second line, the literal string "boolean" is dereferenced using * before the comparison.
The second line is more idiomatic and performant in Rust, as it avoids the unnecessary creation of a String object and relies on the compiler's ability to optimize string literal comparisons.